lib/options-manager/index.js

```diff
@@ -135,7 +135,7 @@ function OptionsManager(widget, options) {
   if (this.icon) {
     this.headerIcon = this.icon;
   } else {
-    this.headerIcon = "//cdn.auth0.com/styleguide/1.0.0/img/badge.png";
+    this.headerIcon = "//cdn.auth0.com/styleguide/latest/lib/logos/img/badge.png";
```

Comment:
@ramasilveyra not sure how is versioning being handled on styleguide but is there a way to depend on something else than `latest`?